### PR TITLE
Add one line into covid tracking states statvars mcf

### PR DESCRIPTION
**Pull request by wh1210**

### CODE REVIEW

File: schema/StatisticalVariable.mcf [hunk 1]
```diff
@@ -179,3 +179,99 @@ medicalStatus: dcs:PatientDeceased
...
+measuredProperty: dcs:count
+medicalStatus: dcs:PatientOnVentilator
+
+Node: dcid:CumulativeCount_MedicalConditionIncident_COVID_19_PatientOnVentilator
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+medicalCondition: dcs:COVID_19
+statType: dcs:measuredValue
+measuredProperty: dcs:cumulativeCount
+medicalStatus: dcs:PatientOnVentilator
```
Comment (tjann) [inline schema/StatisticalVariable.mcf:277]: We decided not to add the new StatisticalVariables to the schema folder. These are all the same as the ones in your import subdir, correct? If so, we can just undo these changes.

File: scripts/covid_tracking_project/historic_state_data/config.txt [hunk 2]
```diff
@@ -3,12 +3,12 @@ import: {
...
+    mcf_url: "gs://resolved_mcf/external_tables/covid_tracking_states_new/nodes.mcf-00000-of-00001"
     # First date of imported data.
     start_date: "2020-01-22"
     # Last date of imported data.
-    end_date: "2020-06-22"
+    end_date: "2020-06-25"
     # Source data release schedule.
     update_frequency: "P1D"
-    next_release_date: "2020-06-23"
+    next_release_date: "2020-06-26"
```
Comment (tjann) [inline scripts/covid_tracking_project/historic_state_data/config.txt:13]: Can also remove end_date and next_release_date here so that future no need to keep updating this.

File: scripts/covid_tracking_project/historic_state_data/preprocess_csv.py [hunk 3]
```diff
@@ -63,7 +63,7 @@
 variableMeasured: dcs:{stat_var}
 measurementMethod: dcs:CovidTrackingProject
 observationAbout: C:COVIDTracking_States->GeoId
-observationDate: C:COVIDTracking_States->Dates
+observationDate: C:COVIDTracking_States->Date
```
Comment (tjann) [inline scripts/covid_tracking_project/historic_state_data/preprocess_csv.py:66]: Oof, good catch. How did this work before?

Comment (wh1210) [reply scripts/covid_tracking_project/historic_state_data/preprocess_csv.py:66]: No worry about this, I am thinking this is because I unintentionally add a 's' when using 'ctrl + s' to save file. The tmcf in prod is the correct one~~

File: schema/StatisticalVariable.mcf (hunk 1 above)
Comment (wh1210) [reply schema/StatisticalVariable.mcf:277]: Yes, let me do some changes.

File: scripts/covid_tracking_project/historic_state_data/config.txt (hunk 2 above)
Comment (wh1210) [reply scripts/covid_tracking_project/historic_state_data/config.txt:13]: Got it.

File: schema/StatisticalVariable.mcf (hunk 1 above)
Comment (wh1210) [reply schema/StatisticalVariable.mcf:277]: Yep, this is also what I am thinking as this will cause conflicts easily and would potentially cause some problems unintentionally.
